test/test_version.py

```diff
@@ -20,11 +20,11 @@ async def test_different_version(
         name: str = "cve-bin-tool"
 
         # Reducing the version number
-        _version = VERSION.split(".")
-        _version[0] = str(int(_version[0]) + version_jump)
-        _version = ".".join(_version)
+        _versions = VERSION.split(".")  # type: List[str]
+        _versions[0] = str(int(_versions[0]) + version_jump)
+        _version = ".".join(_versions)  # type: str
 
-        # Mocking the get function to return `_version`
+        # Mocking the g/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.htmlet function to return `_version`
```

Comment:
Not sure what happened here... accidental tab complete? Needs fixing, anyhow

Reply:
Apologies for that, it is fixed now